index.html

```diff
@@ -1,111 +1,579 @@
+              >
+              <a
+                href="https://www.LinkedIn.com/yourpage"
+                target="_blank"
+                rel="noopener noreferrer"
+                >LinkedIn</a
+              >
+              <a
+                href="https://www.Instagram.com/yourpage"
+                target="_blank"
+                rel="noopener noreferrer"
+                >Instagram</a
+              >
+            </div>
+          </div>
+          <div>
+            <img src="#" alt="text" />
+            <h3>Jason Smith</h3>
+            <p>Co-Founder</p>
+            <div>
+              <a
+                href="https://www.Twitter.com/yourpage"
+                target="_blank"
+                rel="noopener noreferrer"
+                >Twitter</a
+              >
+              <a
+                href="https://www.facebook.com/yourpage"
+                target="_blank"
+                rel="noopener noreferrer"
+                >Facebook</a
+              >
+              <a
+                href="https://www.Telegram.com/yourpage"
+                target="_blank"
+                rel="noopener noreferrer"
+                >Telegram</a
+              >
+              <a
+                href="https://www.LinkedIn.com/yourpage"
+                target="_blank"
+                rel="noopener noreferrer"
+                >LinkedIn</a
+              >
+              <a
+                href="https://www.Instagram.com/yourpage"
+                target="_blank"
+                rel="noopener noreferrer"
+                >Instagram</a
+              >
             </div>
           </div>
         </div>
-      </template>
-      <!-- <script src="script.js"></script> --> <!-- uncomment script --> 
-</body>
+      </section>
+      <section class="faqs">
+        <h2>FAQs</h2>
+        <div class="details">
+          <details>
+            <h3>What are the NTFs?</h3>
+            <button type="button">+</button>
+          </details>
+          <summary>
+            Urna vitae erat et lacus, consectetur ac nulla vestibulum lobortic.
+            Nula dapibus urna volutpat vestibulum lobortic. Nulla dapibus urna
+            volutpat venenatis, risus faucibus.
+          </summary>
+          <div>
+            <h3>How do i get NTFs?</h3>
+            <button type="button">+</button>
+          </div>
+          <div>
+            <h3>How can we buy your NTFs?</h3>
+            <button type="button">+</button>
+          </div>
+          <div>
+            <h3>Who are the team behind the project?</h3>
+            <button type="button">+</button>
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer>
+      <div>
+        <div class="footer-logo">
+          <a href="#">Binabox</a>
+          <p>We are the best way to check the rarity of NFT collection.</p>
+        </div>
+        <div class="menu">
+          <ul>
+            <li>
+              <a href="#" class="menu">Subscribe</a>
```

Comment:
Please note that the parent and child tags have the same classes. This is not good. It's better to note this dependency in the class name. For example, if the parent class is named 'menu', then the child class can be named 'menu-item' or 'menu-link'

Reply:
done